services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/allu/ApplicationPdfServiceTest.kt

```diff
@@ -0,0 +1,331 @@
+                                    orderer = true
+                                ),
+                            ),
+                    representativeWithContacts =
+                        AlluDataFactory.createPersonCustomer(
+                                name = "Reynold Representative",
+                                registryKey = "281192-937W",
+                                email = "reynold@company.test",
+                                phone = "0509990000",
+                            )
+                            .withContacts(),
+                    propertyDeveloperWithContacts =
+                        AlluDataFactory.createCompanyCustomer(
+                                name = "Developer Inc.",
+                                registryKey = "8545758-6",
+                                email = "info@developer.test",
+                                phone = "0508888888",
+                            )
+                            .withContacts(
+                                AlluDataFactory.createContact(
+                                    name = "Denise Developer",
+                                    email = "denise@developer.test",
+                                    phone = "0502222222"
+                                ),
+                            ),
+                )
+
+            val pdfData = ApplicationPdfService.createPdf(applicationData, 614f, listOf())
+
+            assertThat(getPdfAsText(pdfData)).all {
+                contains("Company Ltd")
+                contains("1054713-0")
+                contains("info@company.test")
+                contains("050123456789")
+                contains("Cole Contact")
+                contains("cole@company.test")
+                contains("050987654321")
+                contains("Seth Secondary")
+                contains("seth@company.test")
+                contains("0505556666")
+                contains("Contractor Inc.")
+                contains("0156555-6")
+                contains("info@contractor.test")
+                contains("0509999999")
+                contains("Cody Contractor")
+                contains("cody@contractor.test")
+                contains("0501111111")
+                contains("Reynold Representative")
+                contains("281192-937W")
+                contains("reynold@company.test")
+                contains("0509990000")
+                contains("Developer Inc.")
+                contains("8545758-6")
+                contains("info@developer.test")
+                contains("0508888888")
+                contains("Denise Developer")
+                contains("denise@developer.test")
+                contains("0502222222")
+            }
+        }
+    }
+
+    private fun getPdfAsText(pdfData: ByteArray): String {
+        val reader = PdfReader(pdfData)
+        val pages = reader.numberOfPages
+        val textExtractor = PdfTextExtractor(reader)
+        File("pdfTest.pdf").writeBytes(pdfData)
```

Comment:
<img width="422" alt="image" src="https://user-images.githubusercontent.com/78962935/221501692-e5dfdc81-d89e-44ef-aa7c-1543064c77b2.png">

This seems to actually create a pdf in hanke-service. I imagine this is not wanted.

Reply:
I used this as a way to see what the PDF looks like in different situations while developing and forgot to remove it. Removed it now. Thank you for noticing.

Clearly I didn't check my own code when creating the PR. I'll do a full read-through to see if there are any other bits that were committed by accident.

Reply:
There was one more thing: wiretapping and logging all HttpClient requests. I used at some point to verify the communication with Allu and forgot to remove it. Removed in https://github.com/City-of-Helsinki/haitaton-backend/pull/264/commits/61895f0e30daabc1087907e00748f2bcf7601048.